ci-operator/config/openstack-k8s-operators/openstack-ansibleee-operator/openstack-k8s-operators-openstack-ansibleee-operator-main.yaml

```diff
@@ -68,7 +69,7 @@ tests:
     git log -1
     mkdir -p ../operator && cp -r . ../operator
     cd ../operator
-    pre-commit run --all-files --show-diff-on-failure
+    BUNDLE_PATH="./local/bundle" pre-commit run --all-files --show-diff-on-failure
```

Comment:
Should not this be `/usr/bin/bundle` ?
Error says `Bundler not found. On Linux run 'sudo dnf install /usr/bin/bundle' to install it.`

Reply:
actually, `/usr/bin/bundle` is for installer the bundler, but it depends on which ruby version you get.
by `dnf install -y rubygem-bundler` I'm ensuring the bundler is installed

Reply:
Ok, $BUNDLE_PATH is the location on disk where all gems in your bundle will be located. I've forgotten ruby bits now :smile: